WeakAuras/WeakAuras.lua

```diff
@@ -5554,3 +5572,23 @@ function WeakAuras.FindUnusedId(prefix)
   end
   return id
 end
+
+function WeakAuras.SerializeTable(t)
+  local s = {"{"}
+  for k, v in pairs(t) do
+    s[#s+1] = k
+    s[#s+1] = "="
+    if type(v) == "boolean" then
+      s[#s+1] = v and "true" or "false"
+    elseif type(v) == "string" then
+      s[#s+1] = ('"%s"'):format(v)
```

Comment:
Nope. This will create an invalid string with input that contains a “.

Reply:
will fix